README.md

```diff
@@ -1,31 +1,29 @@
-# GPU Passthrough Manager
 
-[![Showcase Video](https://img.youtube.com/vi/ttMAGxJ4pUo/0.jpg)](https://youtu.be/ttMAGxJ4pUo)
```

Comment:
You can add the showcase video back, I didn't think it was very necessary but it's up to you